run/send-push-message

```diff
@@ -28,7 +28,7 @@ function send_discord() {
   log "Sending Discord message."
 
   curl -H "Content-Type: application/json" -d \
-    '{"username": "'"$title"'", "content": "'"$message"'"}' \
+    '{"username": "'"$DISCORD_USERNAME"'", "content": "'"$message"'"}' \
```

Comment:
This will break existing users that do an upgrade, since they won't know to add the new variable to their config file.

Reply:
Since the "title" field is used for other notification targets too, I decided to make this more general: https://github.com/marcone/teslausb/commit/5d363b99263c2c54e772607b0928f9b6836cdd4b